client/internal/routemanager/manager.go

```diff
@@ -144,8 +146,6 @@ func (m *DefaultManager) Init() (nbnet.AddHookFunc, nbnet.RemoveHookFunc, error)
 		return nil, nil, fmt.Errorf("setup routing: %w", err)
```

Comment:
What will happen if an issue occurs? Does the loaded state cause future issues?

Reply:
If we don't init the route selector before returning here it will be nil and we'll panic later